docs/sources/flow/reference/components/otelcol.processor.attributes.md

```diff
@@ -0,0 +1,582 @@
+`services` | `list(string)` | A list of items to match the service name against. | `[]` | no
+`span_names` | `list(string)` | A list of items to match the span name against. | `[]` | no
+`log_bodies` | `list(string)` | A list of strings that the LogRecord's body field must match against. | `[]` | no
+`log_severity_texts` | `list(string)` | A list of strings that the LogRecord's severity text field must match against. | `[]` | no
+`metric_names` | `list(string)` | A list of strings to match the metric name against. | `[]` | no
+`span_kinds` | `list(string)` | A list of items to match the span kind against. | `[]` | no
+
+`match_type` is required and can be set to either `"regexp"` or `"strict"`.
+
+One of the following is also required:
+* For spans, one of `services`, `span_names`, `span_kinds`, [attribute], [resource], or [library] must be specified with a non-empty value for a valid configuration. The `log_bodies`, `log_severity_texts`, `resource_attributes` and `metric_names` fields are invalid.
+* For logs, one of `log_bodies`, `log_severity_texts`, [attribute], [resource], or [library] must be specified with a non-empty value for a valid configuration. The `span_names`, `span_kinds`, `metric_names`, `resource_attributes`, and `services` fields are invalid.
+* For metrics, one of `metric_names`, [resource] must be specified with a valid non-empty value for a valid configuration. The `span_names`, `span_kinds`, `log_bodies`, `log_severity_texts` and `services` fields are invalid.
+
+For `metric_names`, a match occurs if the metric name matches at least one item in the list.
+For `span_kinds`, a match occurs if the span's span kind matches at least one item in this list.
+
+Note: If both `include` and `exclude` are specified, the include properties are checked before the exclude properties.
+
+### regexp block
+
+This block is optional configuration for the `match_type` of `"regexp"`.
+
+The following arguments are supported:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`cacheenabled` | `bool` | Determines whether match results are LRU cached. | `false` | no
+`cachemaxnumentries` | `int` | The max number of entries of the LRU cache that stores match results. | `0` | no
+
+Enabling `cacheenabled` could make subsequent matches faster.
+Cache size is unlimited unless `cachemaxnumentries` is also specified.
+
+`cachemaxnumentries` is ignored if `cacheenabled` is false.
+
+### attribute block
+
+This block specifies a list of attributes to match against.
+Only `match_type = "strict"` is allowed if `attribute` is specified.
+
+The following arguments are supported:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`key` | `string` | The attribute key. | | yes
+`value` | `any` | The value to match against. | | no
+
+If `value` is not set, any value will match.
+The type of `value` could be a number, a string or a boolean.
+
+### resource block
```

Comment:
This is the same as the attribute block. Should I repeat it, or try have it under the same heading ("attributes/resource block")?

Reply:
No, don't merge blocks please, even if they have the same arguments.